GraphConsoleAppV3/Constants.cs

```diff
@@ -2,12 +2,26 @@
 {
     internal class Constants
     {
-        public const string TenantName = "GraphDir1.onMicrosoft.com";
-        public const string TenantId = "4fd2b2f2-ea27-4fe5-a8f3-7b1a7c975f34";
-        public const string ClientId = "118473c2-7619-46e3-a8e4-6da8d5f56e12";
-        public const string ClientSecret = "hOrJ0r0TZ4GQ3obp+vk3FZ7JBVP+TX353kNo6QwNq7Q=";
-        public const string ClientIdForUserAuthn = "66133929-66a4-4edc-aaee-13b04b03207d";
-        public const string AuthString = "https://login.microsoftonline.com/" + TenantName;
+        public const string TenantNameAppOnly = "calvliTest.onMicrosoft.com";
+        public const string TenantIdAppOnly = "2ea86d85-3149-476d-bd4a-1cc746ef2f01";
+        public const string ClientId = "a7ec6203-e14c-43c9-9369-c044476a3be0";
+        public const string ClientSecret = "eeveaj0RSnQRheEdzVDqQcOAAibL1oJxJs7zhgMCypQ=";
+        //public const string ClientIdForUserAuthn = "66133929-66a4-4edc-aaee-13b04b03207d";
+        public const string AuthStringAppOnly = "https://login.microsoftonline.com/" + TenantNameAppOnly;
+        //public const string ResourceUrl = "https://graph.windows.net";
+
+        //PROD
+        //public const string TenantName = "adatumisv.onmicrosoft.com";
+        public const string TenantId = "2ea86d85-3149-476d-bd4a-1cc746ef2f01";
+        public const string TenantName = "Calvin Li";
+        //public const string TenantId = "a830edad9050849NDA1.onmicrosoft.com";
+
+        //public const string ClientId = "27dd62a3-ba34-4833-b5c4-5698457436c1";
```

Comment:
I think we should add constants.cs to .gitIgnore. Afterwards, we can make a one-time edit to remove the comments.